app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    num_parts = 5
+    res = split_integer(value, num_parts)
+    assert sum(res) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    expected = split_integer(10, 5)
+    result = [2, 2, 2, 2, 2]
+    assert expected == result
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    expected = split_integer(2, 1)
+    result = [2]
+    assert expected == result
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    expected = split_integer(2, 3)
+    result = sorted(expected)
+    assert result == expected
```

Comment:
The test `test_parts_should_be_sorted_when_they_are_not_equal` is not correctly verifying the sorting of the parts. The assertion `assert result == expected` will always pass because `result` is the sorted version of `expected`. Instead, you should compare `expected` with a manually sorted list to ensure the function returns a sorted list. For example, `assert expected == [0, 1, 1]` if that's the expected sorted output.